### ISSUE TITLE
Images.java:37-38: Keep implementing the rest of the...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#98]
The puzzle `83-f448d301` from #83 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/bc8e27a93cff11821587f082de9b238e0a506af6/src/main/java/com/amihaiemil/docker/Images.java#L37-L38

The puzzle was created by Mihai A on 07-May-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/G6LPQQV2P) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#98](https://github.com/amihaiemil/docker-java-api/issues/98) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#98](https://github.com/amihaiemil/docker-java-api/issues/98) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1301333166); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@0crat waiting on review/merge of #106

**Comment by 0crat** [comment]
> @0crat waiting on review/merge of #106 ([here](https://github.com/amihaiemil/docker-java-api/issues/98#issuecomment-388958410))

@llorllale The impediment for [#98](https://github.com/amihaiemil/docker-java-api/issues/98) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `83-f448d301` has disappeared from the source code, that's why I closed this issue. //cc @amihaiemil

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#98](https://github.com/amihaiemil/docker-java-api/issues/98) is now out of scope
